libs/vkd3d/device.c

```diff
@@ -80,12 +80,12 @@ static const struct vkd3d_optional_extension_info optional_device_extensions[] =
     VK_EXTENSION(KHR_SHADER_MAXIMAL_RECONVERGENCE, KHR_shader_maximal_reconvergence),
     VK_EXTENSION(KHR_SHADER_QUAD_CONTROL, KHR_shader_quad_control),
     VK_EXTENSION(KHR_COMPUTE_SHADER_DERIVATIVES, KHR_compute_shader_derivatives),
+    VK_EXTENSION(KHR_CALIBRATED_TIMESTAMPS, KHR_calibrated_timestamps),
```

Comment:
Should we not check both? The KHR one is fairly recent.

Reply:
Seems like the cutoff is Mesa 24.0 and NV 535. Maybe that's okay?

Reply:
Can we confirm that stable Deck ships the KHR?

Reply:
vulkaninfo says yes.

On the Nvidia side we kinda dropped <535 already anyway, so I don't think that's a concern here. The KHR is like one and a half years old already anyway.